boring/src/ssl/mod.rs

```diff
@@ -57,6 +57,20 @@
 //!     }
 //! }
 //! ```
+
+// Clippy doesn't like `&` masking with 0. This makes sense, because a mask of 0
+// always clears the entire value, so the mask expression could just be the
+// literal `0`. However, `& 0` shows up in the code generated by the `bitflags!`
+// macro --- presumably, this is because it's simpler to have the macro generate
+// expressions like `bits & 1 << 2`, `bits & 1 << 1`, and finally `bits & 1 <<
+// 0` or similar, rather than special-casing the 0 case. Thus, allow this lint
+// in this module.
+//
+// Unfortunately, we can't put the `allow` attribute on the `bitflags!` type
+// itself, because the macro doesn't accept arbitrary attributes on the
+// generated type. So, we have to allow this for the whole module.
+#![allow(clippy::bad_bit_mask)]
```

Comment:
Just fixed it in context of https://github.com/cloudflare/boring/pull/160, the fix is to update bitflags to ^2.0

Reply:
I think we can just merge #160 and just rebase your previous PR on top of it

Reply:
sounds good to me --- happy to drop that part of the PR!

Reply:
@inikulin let me know when I should rebase?

Reply:
#160 was merged